src/Components/EntityHeaders/EntityHeaderSubmission.tsx

```diff
@@ -0,0 +1,85 @@
+import ChevronRightIcon from "@artsy/icons/ChevronRightIcon"
+import { Avatar, BoxProps, Flex, Text } from "@artsy/palette"
+import { EntityHeaderSubmission_submission$data } from "__generated__/EntityHeaderSubmission_submission.graphql"
+import { FC } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface EntityHeaderSubmissionProps extends BoxProps {
+  submission: EntityHeaderSubmission_submission$data
+  displayAvatar?: boolean
+  displayLink?: boolean
+  FollowButton?: JSX.Element
+}
+
+const EntityHeaderSubmission: FC<EntityHeaderSubmissionProps> = ({
+  submission,
+  displayAvatar = true,
+  displayLink = true,
+  FollowButton,
+  ...rest
+}) => {
+  const artist = submission.artist
+
+  if (!artist) return null
```

Comment:
If the artist is null/undefined, would we still be able to proceed with editing and saving the submission?